frontend/src/scenes/retention/RetentionModal.tsx

```diff
@@ -39,136 +38,121 @@ export function RetentionModal({
-                                                        <Link
-                                                            to={urls.group(
-                                                                String(personAppearances.person.group_type_index),
-                                                                personAppearances.person.group_key
-                                                            )}
-                                                            data-attr="retention-person-link"
-                                                        >
-                                                            {groupDisplayId(
-                                                                personAppearances.person.group_key,
-                                                                personAppearances.person.properties
-                                                            )}
-                                                        </Link>
-                                                    ) : (
-                                                        <Link
-                                                            to={urls.person(personAppearances.person.distinct_ids[0])}
-                                                            data-attr="retention-person-link"
-                                                        >
-                                                            {asDisplay(personAppearances.person)}
-                                                        </Link>
-                                                    )}
-                                                </td>
-                                                {personAppearances.appearances.map(
-                                                    (appearance: number, index: number) => {
-                                                        return (
-                                                            <td
-                                                                key={index}
-                                                                className={
-                                                                    appearance
-                                                                        ? 'retention-success'
-                                                                        : 'retention-dropped'
-                                                                }
-                                                            />
+            <div className="min-h-20">
+                {actorsLoading ? (
+                    <SpinnerOverlay />
+                ) : results[selectedRow]?.values[0]?.count === 0 ? (
+                    <span>No {aggregationTargetLabel.plural} during this period.</span>
+                ) : (
+                    <>
+                        <table className="RetentionTable">
+                            <tbody>
+                                <tr>
```

Comment:
![Screenshot 2022-09-05 at 14 51 12](https://user-images.githubusercontent.com/984817/188465115-4bed187c-c521-428a-86a3-be1055c58931.png)

I can export myself here but I don't show in the modal

Reply:
Hmmmmm

Reply:
I can't replicate this 😅 might ping you for a pairing session to see if we can figure out whats happening

Reply:
So I could replicate this but it happens on master as well. Could be related to the "missing_persons" stuff that @neilkakkar looked at.

Reply:
What's the core problem here?

Reply:
Hard to say but it just seems we could get into a state where the results mention >= 1 people but the actual API returns nothing. Couldn't see an obvious reason why but made me think of the `missing_persons` stuff you added. Could it be the same thing?